pkg/registry/core/service/ipallocator/allocator.go

```diff
@@ -254,7 +254,7 @@ func calculateIPOffset(base *big.Int, ip net.IP) int {
 func RangeSize(subnet *net.IPNet) int64 {
```

Comment:
My vote would be to address it now, since the submit queue is blocked. BTW the current assignment has @bowei, instead of MrHohn.